### DISCUSSION TITLE
GPS Animal

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#528] [Curiosidades]
Post de referência: https://www.vish.blog.br/curiosidades/gps-animal/

**Comment by brunoaugusto** [comment]
"Who's your 'fuend'?" Quem dublou esse rato, o Kripke? :joy:

**Comment by angellisy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/curiosidades/gps-animal/#comment_1160603543>):

> "Who's your 'fuend'?" Quem dublou esse rato, o Kripke? 😂

![null](<https://i.imgur.com/r4Q7dpc.png>)
